client/src/components/ui/table/themes.ts

```diff
@@ -0,0 +1,17 @@
+import { createTheme } from "react-data-table-component";
+
+export const overrideDarkMode = () =>
```

Comment:
Is dit de enige theme?
Waarom is dit expliciet zo gedefinieerd?
Of gewoon netter en misschien ooit nuttig?

Reply:
Ik dacht eigenlijk dat ik meer met het theme zou kunnen, maar veel dingen gaan er niet mee. Misschien komt er ooit nog een ander theme maar voorlopig niet